editor/js/Editor.js

```diff
@@ -424,6 +424,10 @@ Editor.prototype = {
 
 					helper = new THREE.SkeletonHelper( object.skeleton.bones[ 0 ] );
 
+				} else if ( object.isBone && ( !object.parent || !object.parent.isBone ) ) {
```

Comment:
I think this can be simplified like so:
```js
} else if ( object.isBone && ( object.parent?.isBone !== true ) ) {
```

Reply:
BTW: The policy is to test for `is*` flags like so:
```js
console.log( object.isBone === true ); // detecting all bones
console.log( object.isBone !== true ); // detecting all 3D object except for bones
```

Reply:
Thanks! Updated.